packages/colorpickers/src/elements/ColorpickerDialog/index.tsx

```diff
@@ -64,6 +68,12 @@ export interface IColorpickerDialogProps extends IColorpickerProps {
    * Passes HTML attributes to the color dialog button element
    */
   buttonProps?: HTMLAttributes<HTMLButtonElement>;
+  /**
+   * Handles dialog changes
+   *
+   * @param {object} changes The changed dialog state
```

Comment:
#nitpick here, should the param description include `isOpen` property on the object?